### PR TITLE
fix: some react warnings

### PR DESCRIPTION
**Pull request by MatissJanis**
Fixing two react warnings. See inlined comments for more details.

### CODE REVIEW

File: packages/loot-design/src/components/budget/rollover/rollover-components.js [hunk 1]
```diff
@@ -426,7 +426,6 @@ export function IncomeCategoryMonth({
         name="received"
         width="flex"
         borderColor={colors.border}
-        textAlign="right"
```
Comment (MatissJanis) [inline packages/loot-design/src/components/budget/rollover/rollover-components.js:429]:
```
index.js:1451 Warning: React does not recognize the `textAlign` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `textalign` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
    in div (at View.js:13)
    in View (at table.js:93)
    in ForwardRef(Field) (at rollover-components.js:425)
```

File: packages/loot-design/src/components/common.js [hunk 2]
```diff
@@ -111,18 +111,27 @@ export function AnchorLink({
...
+  ({ asAnchor, children, ...props }, ref) => {
+    function onClick(e) {
+      e.preventDefault();
+      window.Actual.openURLInBrowser(props.href);
+    }
 
-  if (props.asAnchor) {
-    // eslint-disable-next-line
-    return <a ref={ref} {...props} onClick={onClick} />;
+    if (asAnchor) {
```
Comment (MatissJanis) [inline packages/loot-design/src/components/common.js:121]:
```
index.js:1451 Warning: React does not recognize the `asAnchor` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `asanchor` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
    in a (at common.js:122)
    in ForwardRef (at WelcomeScreen.js:27)
    in div (at common.js:642)
    in P (at WelcomeScreen.js:25)
```

Comment (MatissJanis) [reply packages/loot-design/src/components/common.js:121]: Functionally this component has really no changes. It just looks like a lot due to prettier re-formating things.

**Review by trevdor — APPROVED** [review]
Nice! Thanks for cleaning these up.

**Pull request merged**